### DISCUSSION TITLE
Is S/be commonly known ?

### DISCUSSION THREAD

**Discussion by cverbiest** [Riverside-Software/pct#441] [General]
@PeterJudge-PSC Is S/be a common abbreviation, should I add it to the TL/DR AFAIK IMHO list ?

**Comment by gquerret** [comment]
First time I saw S/be :)

**Reply by PeterJudgeZA** [reply]
I'm not sure how common it is, but I like it for subject lines.  In case you care, I have a few others too ...
s/be = should be
s/not = should not
w/<thing> = with <thing>
w/o <thing> = without <thing>

**Reply by cverbiest** [reply]
Thank you for the list, I don't mind learning some new abbreviations.
